### ISSUE TITLE
Keep method arguments same as base class for GPURenderPassEncoder, GPURenderBundleEncoder

### ISSUE THREAD

**Issue by drawmindmap** [gpuweb/types#24]
```
const passEncoder = commandEncoder.beginRenderPass(this.renderPassDescriptor);
passEncoder.draw(3);
```
Typescript complain:
passEncoder.draw expected 4 arguments, but got 1.
An argument for 'instanceCount' was not provided.

**Comment by kainino0x** [comment]
Thanks for catching this!

**Comment by kainino0x** [comment]
Unfortunate that this wasn't caught by TypeScript - I would argue that there's an unsoundness bug here (the "implements" should have been unsatisfied). Couldn't find any info online about why it behaves this way.

Anyway, published v0.0.29
